data/campaigns/World_Conquest/lua/map/postgeneration_utils/engine.lua

```diff
@@ -76,7 +76,7 @@ function set_terrain_impl(data)
 end
 
 function set_terrain_simul(cfg)
-	cfg = wml.parsed(cfg)
+	cfg = cfg
```

Comment:
Surely the line can be removed now.

Reply:
This can indeed be removed. I shall wait for any more requested changes and then make them all in one single commit.

Reply:
It looks like this line was incorrect to begin with - the `cfg` isn't even a config at all. So, simply removing it should be the correct answer.

Even if it stilll worked _with_ this line, it was just a fluke; this isn't a supported use of `wml.parsed`. Maybe we should add a warning in case it's misused like this elsewhere.

Reply:
This change has been done in the second commit.